pandas/tests/io/parser/test_c_parser_only.py

```diff
@@ -44,32 +44,6 @@ def test_buffer_overflow(c_parser_only, malformed):
         parser.read_csv(StringIO(malformed))
 
 
-def test_buffer_rd_bytes(c_parser_only):
```

Comment:
Based on the original issue, this sounded like a PY2 bug specifically so I don't think this needs testing anymore